js/Story.js

```diff
@@ -224,50 +220,50 @@ Story.prototype.getTemplateForTiddler = function(title,template,tiddler)
 Story.prototype.refreshTiddler = function(title,template,force,customFields,defaultText)
 {
 	var tiddlerElem = this.getTiddler(title);
-	if(tiddlerElem) {
-		if(tiddlerElem.getAttribute("dirty") == "true" && !force)
-			return tiddlerElem;
-		template = this.chooseTemplateForTiddler(title,template);
-		var currTemplate = tiddlerElem.getAttribute("template");
-		if((template != currTemplate) || force) {
-			var tiddler = store.getTiddler(title);
-			if(!tiddler) {
-				tiddler = new Tiddler();
-				if(store.isShadowTiddler(title)) {
-					var tags = [];
-					tiddler.set(title,store.getTiddlerText(title),config.views.wikified.shadowModifier,version.date,tags,version.date);
-				} else {
-					var text = template==config.tiddlerTemplates[DEFAULT_EDIT_TEMPLATE] ? // #166
-								config.views.editor.defaultText.format([title]) :
-								config.views.wikified.defaultText.format([title]);
-					text = defaultText || text;
-					var fields = customFields ? customFields.decodeHashMap() : null;
-					tiddler.set(title,text,config.views.wikified.defaultModifier,version.date,[],version.date,fields);
-				}
-			}
-			tiddlerElem.setAttribute("tags",tiddler.tags.join(" "));
-			tiddlerElem.setAttribute("tiddler",title);
-			tiddlerElem.setAttribute("template",template);
-			tiddlerElem.onmouseover = this.onTiddlerMouseOver;
-			tiddlerElem.onmouseout = this.onTiddlerMouseOut;
-			tiddlerElem.ondblclick = this.onTiddlerDblClick;
-			tiddlerElem[window.event?"onkeydown":"onkeypress"] = this.onTiddlerKeyPress;
-			tiddlerElem.innerHTML = this.getTemplateForTiddler(title,template,tiddler);
-			applyHtmlMacros(tiddlerElem,tiddler);
-			if(store.getTaggedTiddlers(title).length > 0)
-				jQuery(tiddlerElem).addClass("isTag");
-			else
-				jQuery(tiddlerElem).removeClass("isTag");
-			if(store.tiddlerExists(title)) {
-				jQuery(tiddlerElem).removeClass("shadow");
-				jQuery(tiddlerElem).removeClass("missing");
+	if(!tiddlerElem) return;
+	if(tiddlerElem.getAttribute("dirty") == "true" && !force)
```

Comment:
this seems to be a big change, that has nothing to do with the filtering. What does it do?